<?xml version="1.0" encoding="UTF-8"?>
<!--
  - Copyright (c) 2024. aaron.
  -
  - This program is under the GPL-3.0 license.
  - if you have not received it or the program has a bug, please let me know:
  - <communicate_aaron@outlook.com>.
  -->

<ui version="4.0">
    <class>Widget</class>
    <widget class="QWidget" name="Widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>582</width>
                <height>492</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>字符串与数值之间的转换</string>
        </property>
        <widget class="QGroupBox" name="groupBox_number_input_output">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>561</width>
                    <height>231</height>
                </rect>
            </property>
            <property name="title">
                <string>数值输入和输出</string>
            </property>
            <widget class="QLabel" name="label_amount">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>30</y>
                        <width>51</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>数量</string>
                </property>
            </widget>
            <widget class="QComboBox" name="comboBox_amount">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>30</y>
                        <width>141</width>
                        <height>28</height>
                    </rect>
                </property>
            </widget>
            <widget class="QComboBox" name="comboBox_unit_price">
                <property name="geometry">
                    <rect>
                        <x>390</x>
                        <y>30</y>
                        <width>141</width>
                        <height>28</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="label_unit_price">
                <property name="geometry">
                    <rect>
                        <x>330</x>
                        <y>30</y>
                        <width>51</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>单价</string>
                </property>
            </widget>
            <widget class="QComboBox" name="comboBox_total_price">
                <property name="geometry">
                    <rect>
                        <x>390</x>
                        <y>100</y>
                        <width>141</width>
                        <height>28</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="label_total_price">
                <property name="geometry">
                    <rect>
                        <x>330</x>
                        <y>100</y>
                        <width>51</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>总价</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_qdebug_test">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>170</y>
                        <width>101</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>qDebug()测试</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_calculate_total_price">
                <property name="geometry">
                    <rect>
                        <x>330</x>
                        <y>170</y>
                        <width>101</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>计算总价</string>
                </property>
            </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_badecimal_conversion">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>260</y>
                    <width>561</width>
                    <height>211</height>
                </rect>
            </property>
            <property name="title">
                <string>进制转换</string>
            </property>
            <widget class="QLabel" name="label_dec">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>40</y>
                        <width>51</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>十进制</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_bin">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>100</y>
                        <width>51</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>二进制</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_hex">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>160</y>
                        <width>61</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>十六进制</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_dec">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>40</y>
                        <width>251</width>
                        <height>28</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_bin">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>100</y>
                        <width>251</width>
                        <height>28</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="lineEdit_hex">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>160</y>
                        <width>251</width>
                        <height>28</height>
                    </rect>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_dec_conversion">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>40</y>
                        <width>161</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>DEC --&gt; 其他进制</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_bin_conversion">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>100</y>
                        <width>161</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>BIN --&gt; 其他进制</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_hex_conversion">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>160</y>
                        <width>161</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="text">
                    <string>HEX --&gt; 其他进制</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
